example/app/example-app/projections.go

```diff
@@ -150,7 +150,7 @@ func InitProjection(name string, params InitProjectionParams) projection_entity.
 		databaseURL := migrationhelper.GenerateDefaultDatabaseURL(name, connString)
 		migrationHelper := github_migrationhelper.NewGithubMigrationHelper(sourceURL, databaseURL)
 
-		return account.NewAccount(params.Logger, params.RdbConn, params.CosmosAppClient, migrationHelper)
+		return account.NewAccount(params.Logger, params.RdbConn, params.AccountAddressPrefix, params.CosmosAppClient, migrationHelper)
```

Comment:
cosmos app client is no longer needed?

```suggestion
		return account.NewAccount(params.Logger, params.RdbConn, params.AccountAddressPrefix, migrationHelper)
```

Reply:
no longer needed at the moment